packages/nimble-blazor/NimbleBlazor/Components/NimbleAnchor.razor.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Components;
+
+namespace NimbleBlazor;
+
+public partial class NimbleAnchor : ComponentBase
```

Comment:
Can we document these properties? I don't mind if they're essentially a copy/paste of the FAST docs.